WeakAuras/GenericTrigger.lua

```diff
@@ -235,7 +235,11 @@ local function singleTest(arg, trigger, name, value, operator, use_exact)
   elseif (arg.type == "string" or arg.type == "select") then
     return "(".. name .." and "..name.."==" ..(number or ("\""..(tostring(value) or "").."\""))..")";
   elseif (arg.type == "number") then
-    return "(".. name .." and "..name..(operator or "==")..(number or 0) ..")";
+    if numberType == "percent" then
+      return "(duration == 0 or ("..name.." and "..name..(operator or "==")..(number or 0).."* duration".."))";
```

Comment:
This conceptually doesn't work. You are relying on triggers hardcoding a "duration" local, which is not a given, and also means only a single duration can be used per trigger.

Reply:
ah, I see. thanks for reading though!